### PR TITLE
Create zcash-android-pr-triggerbinding.yml

### PR DESCRIPTION
**Pull request by mdr0id**
Adding in triggerbindings

### CODE REVIEW

**Comment by mdr0id** [comment]
Also likely needs https://github.com/Electric-Coin-Company/infrastructure/pull/810

File: ci/tekton/zcash-android-sdk-lint-task.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  resources:
+    inputs:
+      - name: source
+        resourceSpec:
+          type: git
+          params:
+            - name: revision
+              value: master
+            - name: url
+              value: https://github.com/zcash/zcash-android-wallet-sdk
```
Comment (benzcash) [inline ci/tekton/zcash-android-sdk-lint-task.yml:19]: A `Task` just needs to defines the required inputs.

```
    inputs:
      - name: source
        type: git
```

The caller will use refSpec to define the input resource parameters.

Same with the other task.

Check with
```
$ kubectl apply -f ci/tekton/zcash-android-sdk-lint-task.yml 
Error from server (BadRequest): error when creating "ci/tekton/zcash-android-sdk-lint-task.yml": admiss
ion webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming n
ew object: json: unknown field "resourceSpec"
```

**Comment by gmale** [comment]
Once this is merged, does this mean the SDK will build in tekton anytime there is a commit to master?

File: ci/tekton/zcash-android-sdk-lint-task.yml (hunk 1 above)
Comment (mdr0id) [reply ci/tekton/zcash-android-sdk-lint-task.yml]: Is the value something I need in the `zcash-android-pr-triggerbinding.yml`?

I'm confused how it knows where to pull from.

**Comment by mdr0id** [comment]
> Once this is merged, does this mean the SDK will build in tekton anytime there is a commit to master?
Anytime someone submits a PR and an approved user assigns an appropriate github label (e.g. "safe-to-test")

File: ci/tekton/zcash-android-sdk-lint-task.yml (hunk 1 above)
Comment (benzcash) [reply ci/tekton/zcash-android-sdk-lint-task.yml]: https://github.com/zcash/zcash-android-wallet-sdk/pull/137/files#diff-ce8c8f8bcaa48783caf6cee9e5fb6d36R15

That's one of the binding parameters. Github sends you the clone url in the PR post. Everything you need is in the post.

**Comment by gmale** [comment]
@benzcash Can we merge this?

**Comment by benzcash** [comment]
> @benzcash Can we merge this?

Testing the PR with itself :sunglasses: https://tekton-dashboard-prod.z.cash/#/namespaces/default/pipelineruns/zcash-android-pr-pipeline-r5v74

@gmale do you have permission within this GitHub project to add labels like `safe-to-ci` to any PRs?

**Comment by gmale** [comment]
> @gmale do you have permission within this GitHub project to add labels like safe-to-ci to any PRs?

Yes.

**Comment by gmale** [comment]
Seemed to work. Merging.

@benzcash How do I see the output (logs) from those Tekton tasks that ran?

**Pull request merged**